src/Tabs/Tabs.js

```diff
@@ -202,6 +207,7 @@ class Tabs extends Component {
         createElement(tabTemplate || TabTemplate, {
           key: index,
           selected: this.getSelected(tab, index),
+          templateStyles: tabTemplateStyle,
```

Comment:
That's the pattern we use everywhere. I don't think that we should change it now: consistency.